Dockerfile

```diff
@@ -1,27 +1,20 @@
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1
-
+FROM mcr.microsoft.com/dotnet/sdk:5.0 AS BUILDER
 WORKDIR /source
 
 COPY *.csproj .
-
-COPY /FarmCord .
-
 RUN dotnet restore
 
-RUN dotnet build
-
-RUN cd ./bin/Debug/netcoreapp3.1
-
-RUN mkdir FarmCord
-
-RUN cd ./bin/Debug/netcoreapp3.1/FarmCord
+COPY . ./
+RUN dotnet publish -c Release -o out
 
-RUN cp ./FarmCord/creds.json /bin/Debug/netcoreapp3.1/FarmCord/creds.json
 
-RUN mkdir Assets
+FROM mcr.microsoft.com/dotnet/runtime:5.0
+WORKDIR /farm
 
-RUN cp ./FarmCord/Assets/Island.png /bin/Debug/netcoreapp3.1/FarmCord/Assets
+COPY --from=BUILDER /source/out .
 
-RUN mkdir FarmOutput
+# def won't mess anything up :^)
+RUN rm FarmCord
 
-RUN dotnet run
\ No newline at end of file
+COPY FarmCord/creds.json ./FarmCord/
```

Comment:
I know in this case the container is only going to be built for use inside `docker-compose`, but is it really a good idea to be baking the config file into the container when it can just as easily be bind mounted? The advantage there is that you can update it without having to rebuild everything.
I would probably add the mounting of the config file to the `docker-compose.yml` under `volumes`.

Reply:
This would probably also involve adding the config file to `.dockerignore` if done this way.